npm/pkg/dataplane/dataplane.go

```diff
@@ -145,6 +154,29 @@ func (dp *DataPlane) RunPeriodicTasks() {
 		}
 	}()
 
+	go func() {
+		ticker := time.NewTicker(dp.NetPolInterval)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-dp.stopChannel:
+				return
+			case <-ticker.C:
+				// locks policy manager but can be interrupted
+				dp.netPolInfo.Lock()
+				if dp.netPolInfo.numBatches == 0 {
+					dp.netPolInfo.Unlock()
+					return
+				}
+				dp.netPolInfo.numBatches = 0
```

Comment:
why are we setting this to 0 ?

Reply:
now setting num batches to 0 after successful reconcile